python/pyspark/ml/tests/test_tuning.py

```diff
@@ -16,10 +16,8 @@
 #
 
 import tempfile
-import math
 import unittest
 
-import numpy as np
```

Comment:
Ah, sorry for the trouble, but a later commit now needs this numpy import:

```
flake8 checks failed:
./python/pyspark/ml/tests/test_tuning.py:100:16: F405 'np' may be undefined, or defined from star imports: pyspark.ml.tests.test_tuning
        assert np.allclose(avg_metrics, [2.33333333, 2.5, 2.03333333, 5.33333333])
               ^
./python/pyspark/ml/tests/test_tuning.py:101:16: F405 'np' may be undefined, or defined from star imports: pyspark.ml.tests.test_tuning
        assert np.allclose(std_metrics, [0.94280904, 0.40824829, 0.04714045, 1.88561808])
               ^
./python/pyspark/ml/tests/test_tuning.py:143:25: F405 'np' may be undefined, or defined from star imports: pyspark.ml.tests.test_tuning
        self.assertTrue(np.allclose(cvModel.stdMetrics, cvModelCopied.stdMetrics))
                        ^
3     F405 'np' may be undefined, or defined from star imports: pyspark.ml.tests.test_tuning
```

Just don't revert this line

Reply:
Thanks a lot!